com.unity.cinemachine/Samples~/Shared Assets/UI/SampleHelpUI.cs

```diff
@@ -0,0 +1,57 @@
+using System;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+using UnityEngine.UIElements;
+
+namespace Unity.Cinemachine.Samples
+{
+    /// <summary>
+    /// Displays a button in the game view that will bring up a window with scrollable text.
+    /// </summary>
+    [RequireComponent(typeof(UIDocument))]
+    public class SampleHelpUI : MonoBehaviour
+    {
+        public bool VisibleAtStart; // TODO: no sync
+        public string HelpTitle;
+        [TextArea(minLines: 10, maxLines: 50)]
+        public string HelpText;
+
+        VisualElement m_HelpBox, m_HelpToggleBox;
+        Toggle m_HelpToggle;
+
+        void OnEnable()
+        {
+            // try to center vertically too (horizontally it is) alignSelf centered
+            var uiDocument = GetComponent<UIDocument>();
+            
+            m_HelpToggleBox = uiDocument.rootVisualElement.Q("HelpToggleBox");
+            m_HelpToggle = new Toggle("Help");
+            m_HelpToggle.AddToClassList("helpToggle");
+            m_HelpToggle.RegisterValueChangedCallback(evt => m_HelpBox.visible = evt.newValue);
+            m_HelpToggleBox.Add(m_HelpToggle);
+            
+            m_HelpBox = uiDocument.rootVisualElement.Q("HelpTextBox");
+            if (uiDocument.rootVisualElement.Q("HelpTextBox__Title") is Label helpTitle)
+            {
+                if (HelpTitle == string.Empty)
+                    HelpTitle = SceneManager.GetActiveScene().name;
```

Comment:
I wouldn't do it this way, it's not robust.  Leave the HelpTitle setting blank, just put the scene name in the widget.  That way, if the scene is renamed, the title will automatically follow.
Now you'll have to go back and open every scene and make that field blank.  That's why I suggested not upgrading all the scenes until after the PR passes.

Reply:
I don't know what you mean by "just put the scene name in the widget".

I changed the main prefab's Help Title to be blank and then applied it to all children, so I don't have to fix every scene.

Reply:
I mean this:
helpTitle.text = string.IsNullOrEmpty(HelpTitle) ? SceneManager.GetActiveScene().name : HelpTitle;